### ISSUE TITLE
Icons should be tagged with `aria-hidden` if they are purely decorative

### ISSUE THREAD

**Issue by tituspijean** [YunoHost/issues#2739]
cf. https://poliversity.it/@elettrona/115833034638669516

Screen readers will read the emojis, breaking the flow and sense of the page being read.

I propose to update the documentation code of the Buttons and Categories to include seperately the icons, so that they can be output like this: `<span aria-hidden="true">🚀</span>`.
